cadence/freshmint-claim-sale/FreshmintClaimSale.cdc

```diff
@@ -290,6 +342,15 @@ pub contract FreshmintClaimSale {
 
             self.checkAllowlist(address: address)
 
+            let claims = self.claims[address] ?? 0
+
+            // Enforce the claim limit if it is set
+            if let claimLimit = self.claimLimit {
+                assert(claims < claimLimit, message: "reached claim limit")
```

Comment:
Minor / nitpic, could make this assertion after line 352, when you increment the claim count per address. But I think either way works 👍